Removes ssh default and now requires an ssh key arg to build (#20)
rapid7/metasploit-vagrant-builders: README.md

packer init resources/windows/windows.pkr.hcl

# Validate the packer configuration

> packer validate -var "install_pass=${INSTALL_PASS}" -var "authorized_keys_path=./resources/authorized_keys" resources/windows/windows.pkr.hcl

Reviewer: Not a blocker; this is a default now - so it's not really needed

### Security considerations

> The created AMI will require an SSH authorized key to be able to log into the box. If you're a Metasploit developer this will have been made available to you via the password manager. 

Reviewer: ```suggestion
The created AMI will require an SSH authorized key to be able to log into the box - by default this is set to `resources/authorized_keys`. If you are a Rapid7 Metasploit maintainer, pre-existing keys have been available to you via an internal password manager. 
```

Hashicorp does offer [vagrant's "insecure" keypairs](https://github.com/hashicorp/vagrant/tree/9b460ecedefa45a557b1c13c63449839819dc220/keys#insecure-keypairs), which are weak credentials and allow anyone with vagrant's "insecure" keypairs to access the machine. **This is not advised and is not secure**, if this method is chosen you should at the very least behind a restricted security group (i.e. limited to office/host IP addresses).

> Example of adding the keys to the `authorized_keys` file and SSH'ing in via Vagrant's private key, **again this is not secure**:

Reviewer: ```suggestion
Example of adding the keys to the `resources /authorized_keys` file and SSH'ing in via Vagrant's private key, **again this is not secure**:
```

Add the keys:
```bash

> curl https://raw.githubusercontent.com/mitchellh/vagrant/master/keys/vagrant.pub  > authorized_keys

Reviewer: ```suggestion
curl https://raw.githubusercontent.com/mitchellh/vagrant/master/keys/vagrant.pub  > ./resources/authorized_keys
```

```

Example of SSH'ing in via Vagrant's private key:
```bash
curl -L -o ./vagrant_key https://raw.githubusercontent.com/hashicorp/vagrant/main/keys/vagrant
chmod 600 ./vagrant_key

ssh -o PubkeyAcceptedKeyTypes=ssh-rsa -v -i ./vagrant_key vagrant@ec2-54-215-236-141.us-west-1.compute.amazonaws.com
```

> If you're working with a team or company or with a custom box and you want more secure SSH, you should create your own keypair and configure `resources/authorized_keys`.

Reviewer: ```suggestion
```
